src/Tribe/Events/Views/V2/Models/Tickets.php

```diff
@@ -165,6 +165,7 @@ public function __isset( $property ) {
 	 * @since 4.10.9
 	 *
 	 * @since 5.6.3 Add support for the updated anchor link from new ticket templates.
+	 * @since TBD Fixed issue where empty arrays were being returned instead of data.
 	 *
 	 * @return array An array of objects containing the post thumbnail data.
```

Comment:
This is not the correct return note. It's either an empty array or an array with ticket data.
```suggestion
	 * @return array Ticket data or empty array.
```